### PR TITLE
Update Multi-cluster Docs and Images

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
@Sherlock113 I don't understand why there are conflicts [in this commit](https://github.com/kubesphere/website/pull/1636/commits/3d3df07760cfaf2300a0df2345a96bc7b556f73e). Help needed...

File: content/zh/docs/multicluster-management/unbind-cluster.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ weight: 5500
 
 2. 在**集群管理**页面，请点击要从中央控制平面移除的集群。
 
-   ![集群管理](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/cluster-management.PNG)
+   ![集群管理](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/cluster-management.png)
```
Comment (Sherlock113) [inline content/zh/docs/multicluster-management/unbind-cluster.md:22]: Double check in your local build to see if this image is visible.

File: content/zh/docs/multicluster-management/unbind-cluster.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ weight: 5500
...
 
 3. 在**集群设置**下的**基本信息**页面，请选择**我确定要执行解绑集群的操作**，然后点击**解除绑定**。
 
-   ![解绑集群](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/unbind-cluster.PNG)
+   ![解绑集群](/images/docs/zh-cn/multicluster-management/unbind-a-cluster/unbind-cluster.png)
```
Comment (Sherlock113) [inline content/zh/docs/multicluster-management/unbind-cluster.md:26]: Double check in your local build to see if this image is visible.

Comment (Felixnoo) [reply content/zh/docs/multicluster-management/unbind-cluster.md]: All fixed.

**Pull request merged**